extensions/hibernate-orm/deployment/src/test/resources/application-multiple-persistence-units-invalid.properties

```diff
@@ -0,0 +1,7 @@
+quarkus.datasource.users.db-kind=h2
+quarkus.datasource.users.jdbc.url=jdbc:h2:tcp://localhost/mem:users;DB_CLOSE_DELAY=-1
+
+quarkus.hibernate-orm.persistence-units."users".dialect=org.hibernate.dialect.H2Dialect
+quarkus.hibernate-orm.persistence-units."users".log.sql=true
+quarkus.hibernate-orm.persistence-units."users".database.generation=drop-and-create
+quarkus.hibernate-orm.persistence-units."users".datasource=users
```

Comment:
Could you add a comment somewhere to remind why this is an invalid configuration? I guess because it's missing the package names filter?

Reply:
Will do.